/zed/zed_driver:
  ros__parameters:
    autostart: true
    base_link_name: base_link
    camera_fps: 30
    camera_resolution: HD720
    camera_serial_number: -1
    camera_settings_path: ""
    depth_confidence: 20
    depth_fill: false
    depth_max_distance: 5.0
    depth_min_distance: 0.5
    depth_map_filled: true
    depth_mode: ULTRA
    depth_rate: 4
    depth_resolution:
      - 640
      - 360
    depth_stabilization: 100
    depth_texture_confidence: 100
    frame_prefix: ""
    local_frame: odom
    publish_tf: true
    sensors_sampling_time: 2500
    streaming_adaptative_bitrate: true
    streaming_bitrate: 6000
    streaming_chunk_size: 16084
    streaming_codec: ""
    streaming_gop_size: 30
    streaming_sender_ip: ""
    streaming_sender_port: 30000
    streaming_target_fps: 30
    svo_compression: H264
    svo_file: ''
    svo_record: false
    svo_transcode_stream: false
    tracking_depth_min_range: -1.0
    tracking_enable: true
    tracking_enable_area_memory: true
    tracking_enable_imu_fusion: true
    tracking_enable_pose_smoothing: true
    tracking_set_gravity_as_origin: true
    use_sim_time: false
    verbose: true
    video_sd_reliable: false
    video_sd_resolution:
      - 640
      - 360
    video_stream_hd: false
    video_stream_rate: 4
    video_stream_record_path: ""
